src/models/syncRule.ts

```diff
@@ -102,6 +125,20 @@ export default class SyncRule {
         });
     }
 
+    public updateEnabled(enabled: boolean): SyncRule {
+        return SyncRule.build({
+            ...this.syncRule,
+            enabled: enabled.toString(),
```

Comment:
You said in the description that booleans cannot be used, and indeed there is no type boolean in the UI of Dhis2. But is this limitation real? at least in user data store I am using booleans with no problems.